### DISCUSSION TITLE
Serene template, can't view/modify Northwind files?

### DISCUSSION THREAD

**Discussion by rchawla-alphabyte** [serenity-is/Serenity#6011] [Help]
I create a new project using the Serene template. I can't help but notice that Northwind module folders are not available, does anyone know how I can make changes to the Northwind modules?

Thanks

**Comment by luongnv6** [comment]
https://github.com/serenity-is/common-features

**Comment by wezmag** [comment]
You can also clone the Serene project with sub modules from GitHub.

` git clone --recurse-submodules https://github.com/serenity-is/Serene.git SereneDemo`
